src/lib/uploadFilePlaceholder.ts

```diff
@@ -0,0 +1,51 @@
+import { Plugin } from "prosemirror-state";
```

Comment:
The entire thing here seems to be unused now it's no longer based on an image?

Reply:
sure will remove this